From: Daniel Leidert (dale) <daniel.leidert@wgdd.de>
Subject: Add 4.1 compat support and FPI fixes

* catalog: Added. Contains 4.1 compat stuff and general FPIs pointing to
  latest DB4 release.
* catalog.xml: Added for same reason.
* docbook-4.0/catalog.xml: Fixed several wrong FPIs refering to "DocBk".
* docbook-4.0/docbook.cat: Fixed several wrong FPIs refering to "DocBk".
* docbook-4.1.2/catalog.xml: Moved compat catalog entries for DB 4.1 to a
  general catalog. Fixed FPI "-//OASIS//ELEMENTS DocBook XML Information
  Pool V4.1.2//EN".

---
 catalog                   | 38 +++++++++++++++++++++++++
 catalog.xml               | 31 ++++++++++++++++++++
 docbook-4.0/catalog.xml   | 72 +++++++++++++++++++++++++++++++++++++++++++++++
 docbook-4.0/docbook.cat   | 39 +++++++++++++++++++++++++
 docbook-4.1.2/catalog.xml | 66 +++++++++++++++++++++++++++++++++++++++++++
 5 files changed, 246 insertions(+)
 create mode 100644 catalog
 create mode 100644 catalog.xml
 create mode 100644 docbook-4.0/catalog.xml
 create mode 100644 docbook-4.0/docbook.cat
 create mode 100644 docbook-4.1.2/catalog.xml

diff --git a/catalog b/catalog
new file mode 100644
index 0000000..d3e078c
--- /dev/null
+++ b/catalog
@@ -0,0 +1,38 @@
+  -- ...................................................................... --
+  -- This catalog was written for the Debian system ....................... --
+  -- ...................................................................... --
+
+  -- ...................................................................... --
+  -- Prefer PUBID to SYSID ................................................ --
+OVERRIDE YES
+
+  -- ...................................................................... --
+  -- Register FPI "-//OASIS//DTD DocBook XML V4.1//EN" pointing to 4.1.2 .. --
+  -- <URL:http://bugs.debian.org/189129> .................................. --
+PUBLIC  "-//OASIS//DTD DocBook XML V4.1//EN"
+        "4.1/docbookx.dtd"
+SYSTEM  "http://www.oasis-open.org/docbook/xml/4.1/docbookx.dtd"
+        "4.1/docbookx.dtd"
+SYSTEM  "http://docbook.org/xml/4.1/docbookx.dtd"
+        "4.1/docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.1//EN"
+        "/usr/share/xml/declaration/xml.dcl"
+
+  -- ...................................................................... --
+  -- Register FPI "-//OASIS//DTD DocBook XML V4//EN" for latest DocBook ... --
+PUBLIC  "-//OASIS//DTD DocBook XML V4//EN"
+        "4/docbookx.dtd"
+SYSTEM  "http://www.oasis-open.org/docbook/xml/4/docbookx.dtd"
+        "4/docbookx.dtd"
+SYSTEM  "http://docbook.org/xml/4/docbookx.dtd"
+        "4/docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4//EN"
+        "/usr/share/xml/declaration/xml.dcl"
+
+  -- ...................................................................... --
+  -- Register FPI "-//OASIS//DTD DocBook XML//EN" for latest DocBook ...... --
+PUBLIC  "-//OASIS//DTD DocBook XML//EN"
+        "4/docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML//EN"
+        "/usr/share/xml/declaration/xml.dcl"
+
diff --git a/catalog.xml b/catalog.xml
new file mode 100644
index 0000000..65f179c
--- /dev/null
+++ b/catalog.xml
@@ -0,0 +1,31 @@
+<?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN"
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+
+<!-- ..................................................................... -->
+<!-- This catalog was written for the Debian system. ..................... -->
+<!-- ..................................................................... -->
+
+<!-- The FPI and SYSIDs for 4.1 should point to 4.1.2. ................... -->
+
+<public publicId="-//OASIS//DTD DocBook XML V4.1//EN"
+        uri="4.1/docbookx.dtd"/>
+
+<system systemId="http://www.oasis-open.org/docbook/xml/4.1/docbookx.dtd"
+        uri="4.1/docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.1/docbookx.dtd"
+        uri="4.1/docbookx.dtd"/>
+
+<!-- General FPIs shall point to latest DocBook .......................... -->
+
+<public publicId="-//OASIS//DTD DocBook XML V4//EN"
+        uri="4/docbookx.dtd"/>
+
+<public publicId="-//OASIS//DTD DocBook XML//EN"
+        uri="4/docbookx.dtd"/>
+
+</catalog>
+
diff --git a/docbook-4.0/catalog.xml b/docbook-4.0/catalog.xml
new file mode 100644
index 0000000..aecf9c9
--- /dev/null
+++ b/docbook-4.0/catalog.xml
@@ -0,0 +1,72 @@
+<?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN" 
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+
+<!-- ...................................................................... -->
+<!-- XML Catalog data for DocBook XML V4.0............................... -->
+<!-- File catalog.xml ..................................................... -->
+
+<!-- Please direct all questions, bug reports, or suggestions for
+     changes to the docbook@lists.oasis-open.org mailing list. For more
+     information, see http://www.oasis-open.org/.
+  -->
+
+<!-- This is the catalog data file for DocBook V4.0. It is provided as
+     a convenience in building your own catalog files. You need not use
+     the filenames listed here, and need not use the filename method of
+     identifying storage objects at all.  See the documentation for
+     detailed information on the files associated with the DocBook DTD.
+     See XML Catalogs at http://www.oasis-open.org/committees/entity/ for
+     detailed information on supplying and using catalog data.
+  -->
+
+<!-- ...................................................................... -->
+<!-- DocBook driver file .................................................. -->
+
+<public publicId="-//Norman Walsh//DTD DocBk XML V4.0//EN"
+        uri="docbookx.dtd"/>
+
+<!-- Other ways to refer to DocBook XML v4.0............................... -->
+<public publicId="-//Norman Walsh//DTD DocBook XML V4.0//EN"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://www.oasis-open.org/docbook/xml/4.0/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<!-- ...................................................................... -->
+<!-- DocBook modules ...................................................... -->
+
+<public publicId="-//Norman Walsh//ENTITIES DocBook XML Notations V4.0//EN"
+        uri="dbnotnx.mod"/>
+
+<public publicId="-//Norman Walsh//ENTITIES DocBook XML Character Entities V4.0//EN"
+        uri="dbcentx.mod"/>
+
+<public publicId="-//Norman Walsh//ELEMENTS DocBook XML Information Pool V4.0//EN"
+        uri="dbpoolx.mod"/>
+
+<public publicId="-//Norman Walsh//ELEMENTS DocBook XML Document Hierarchy V4.0//EN"
+        uri="dbhierx.mod"/>
+
+<public publicId="-//Norman Walsh//ENTITIES DocBook XML Additional General Entities V4.0//EN"
+        uri="dbgenent.mod"/>
+
+<public publicId="-//Norman Walsh//DTD CALS Table Model XML V4.0//EN"
+        uri="calstblx.dtd"/>
+
+<!-- UNOFFICIAL conversion of the exchange table model to XML, based on the OASIS Exchange -->
+<!--    Table Model PUBLIC "-//SGML Open//DTD Exchange Table Model 19960430//EN"           -->
+<public publicId="-//Normal Walsh//Exchange Table Model 19960430 XML V4.0//EN"
+        uri="soextblx.dtd"/>
+
+<!-- the following FPI is declared for the DTD inside the DTD, but we'll -->
+<!-- use the version from the latest DocBook
+<public publicId="-//OASIS//DTD XML Exchange Table Model 19990315//EN"
+        uri="soextblx.dtd"/> -->
+
+<!-- End of catalog data for DocBook XML V4.0 ............................. -->
+<!-- ...................................................................... -->
+
+</catalog>
diff --git a/docbook-4.0/docbook.cat b/docbook-4.0/docbook.cat
new file mode 100644
index 0000000..17478a2
--- /dev/null
+++ b/docbook-4.0/docbook.cat
@@ -0,0 +1,39 @@
+  -- ...................................................................... --
+  -- Catalog data for DocBook XML V4.0 .................................... --
+  -- ...................................................................... --
+
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
+  -- ...................................................................... --
+  -- DocBook driver file .................................................. --
+
+PUBLIC "-//Norman Walsh//DTD DocBk XML V4.0//EN" "docbookx.dtd"
+SYSTEM "http://www.oasis-open.org/docbook/xml/4.0/docbookx.dtd" "docbookx.dtd"
+SYSTEM "http://docbook.org/xml/4.0/docbookx.dtd" "docbookx.dtd"
+DTDDECL "-//Norman Walsh//DTD DocBk XML V4.0//EN" "/usr/share/xml/declaration/xml.dcl"
+
+  -- ...................................................................... --
+  -- Other ways to refer to DocBook XML v4.0............................... --
+
+PUBLIC "-//Norman Walsh//DTD DocBook XML V4.0//EN" "docbookx.dtd"
+DTDDECL "-//Norman Walsh//DTD DocBook XML V4.0//EN" "/usr/share/xml/declaration/xml.dcl"
+
+  -- ...................................................................... --
+  -- DocBook modules ...................................................... --
+
+PUBLIC "-//Norman Walsh//ENTITIES DocBook XML Notations V4.0//EN" "dbnotnx.mod"
+PUBLIC "-//Norman Walsh//ENTITIES DocBook XML Character Entities V4.0//EN" "dbcentx.mod"
+PUBLIC "-//Norman Walsh//ELEMENTS DocBook XML Information Pool V4.0//EN" "dbpoolx.mod"
+PUBLIC "-//Norman Walsh//ELEMENTS DocBook XML Document Hierarchy V4.0//EN" "dbhierx.mod"
+PUBLIC "-//Norman Walsh//ENTITIES DocBook XML Additional General Entities V4.0//EN" "dbgenent.ent"
+
+-- Unofficial XML version of the CALS Table Model, publicly known as --
+--    PUBLIC "-//USA-DOD//DTD Table Model 951010//EN"                --
+PUBLIC "-//Norman Walsh//DTD CALS Table Model XML V4.0//EN" "calstblx.dtd"
+
+-- UNOFFICIAL conversion of the exchange table model to XML, based on the OASIS Exchange --
+--    Table Model PUBLIC "-//SGML Open//DTD Exchange Table Model 19960430//EN"           --
+PUBLIC "-//Norman Walsh//DTD Exchange Table Model 19960430 XML V4.0//EN" "soextblx.dtd"
+-- the following FPI is declared for the DTD inside the DTD, so I include it             --
+PUBLIC "-//OASIS//DTD XML Exchange Table Model 19990315//EN" "soextblx.dtd"
diff --git a/docbook-4.1.2/catalog.xml b/docbook-4.1.2/catalog.xml
new file mode 100644
index 0000000..4cca270
--- /dev/null
+++ b/docbook-4.1.2/catalog.xml
@@ -0,0 +1,66 @@
+<?xml version='1.0'?>
+<!DOCTYPE catalog PUBLIC "-//OASIS//DTD Entity Resolution XML Catalog V1.0//EN" 
+    "http://www.oasis-open.org/committees/entity/release/1.0/catalog.dtd">
+
+<catalog xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+
+<!-- ...................................................................... -->
+<!-- XML Catalog data for DocBook XML V4.1.2............................... -->
+<!-- File catalog.xml ..................................................... -->
+
+<!-- Please direct all questions, bug reports, or suggestions for
+     changes to the docbook@lists.oasis-open.org mailing list. For more
+     information, see http://www.oasis-open.org/.
+  -->
+
+<!-- This is the catalog data file for DocBook V4.1.2. It is provided as
+     a convenience in building your own catalog files. You need not use
+     the filenames listed here, and need not use the filename method of
+     identifying storage objects at all.  See the documentation for
+     detailed information on the files associated with the DocBook DTD.
+     See XML Catalogs at http://www.oasis-open.org/committees/entity/ for
+     detailed information on supplying and using catalog data.
+  -->
+
+<!-- ...................................................................... -->
+<!-- DocBook driver file .................................................. -->
+
+<public publicId="-//OASIS//DTD DocBook XML V4.1.2//EN"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://www.oasis-open.org/docbook/xml/4.1.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<system systemId="http://docbook.org/xml/4.1.2/docbookx.dtd"
+        uri="docbookx.dtd"/>
+
+<!-- ...................................................................... -->
+<!-- DocBook modules ...................................................... -->
+
+<public publicId="-//OASIS//DTD DocBook CALS Table Model V4.1.2//EN"
+        uri="calstblx.dtd"/>
+
+  <!-- already included in docbook 4.2
+<public publicId="-//OASIS//DTD XML Exchange Table Model 19990315//EN"
+        uri="soextblx.dtd"/> -->
+
+<public publicId="-//OASIS//ELEMENTS DocBook XML Information Pool V4.1.2//EN"
+        uri="dbpoolx.mod"/>
+
+<public publicId="-//OASIS//ELEMENTS DocBook Document Hierarchy V4.1.2//EN"
+        uri="dbhierx.mod"/>
+
+<public publicId="-//OASIS//ENTITIES DocBook Additional General Entities V4.1.2//EN"
+        uri="dbgenent.mod"/>
+
+<public publicId="-//OASIS//ENTITIES DocBook Notations V4.1.2//EN"
+        uri="dbnotnx.mod"/>
+
+<public publicId="-//OASIS//ENTITIES DocBook Character Entities V4.1.2//EN"
+        uri="dbcentx.mod"/>
+
+
+<!-- End of catalog data for DocBook XML V4.1.2 ............................. -->
+<!-- ...................................................................... -->
+
+</catalog>
-- 
2.9.3

